jcg_testcases/src/main/resources/python/AnonymousFunctions.md

```diff
@@ -0,0 +1,81 @@
+# Anonymous Functions
+Anonymous functions, also called Lambda functions, are functions that are not bound to an identifier.
+
+## AF1
+[//]: # (MAIN: global)
+Test the use of an anonymous function as an argument to another function.
+
+```json
+{
+  "directLinks": [
```

Comment:
I wonder where this 'links' terminology stems from. In the Java annotations, we named this 'calls', which seems reasonable to me for call graphs.